distances.py

```diff
@@ -0,0 +1,191 @@
+import numpy as np
+    
+
+def values(hist_dataset, hist_im_query_ini):

```

Comment:
I've changed this function completely so it verifies the size of the histograms to be compared